refinery/data_set_manager/isa_tab_parser.py

```diff
@@ -1173,7 +1168,7 @@ def get_dataset_name(self, path):
         investigation_title = True
         title = None
 
-        with open(investigation_file_name, 'r') as f:
+        with open(investigation_file_name, 'r', encoding='latin-1') as f:
```

Comment:
Why 'latin-1'?

Reply:
i think this is another python hiccup: http://python-notes.curiousefficiency.org/en/latest/python3/text_file_processing.html